NetKAN/RasterPropMonitor-Core.netkan

```diff
@@ -1,51 +0,0 @@
-spec_version: v1.4
-identifier: RasterPropMonitor-Core
-name: RasterPropMonitor Core
-abstract: Plugin and props for use in IVAs
-comment: This package contains the plugin, and the standard parts library.
-author:
-  - Mihara
-  - MOARdV
-  - JonnyOThan
-$kref: '#/ckan/spacedock/2534'
-$vref: '#/ckan/ksp-avc'
-x_netkan_epoch: '1'
-x_netkan_force_v: true
-license: GPL-3.0
-tags:
-  - plugin
-  - crewed
-  - first-person
-depends:
-  - name: ModuleManager
-recommends:
-  - name: DE-IVAExtension
-  - name: FreeIva
-suggests:
-  - name: Astrogator
-  - name: DockingPortAlignmentIndicator
-  - name: NavUtilities
-  - name: SCANsat
-  - name: VesselView
-  - name: ProbeControlRoomRecontrolled
-  - name: kOSPropMonitor
-  - name: Chatterer
-  - name: Reviva
-  - name: MechJeb2
-  - name: RealChute
-  - name: ASETProps
-  - name: ASETAvionics
-  - name: Mk1LanderCanIVAReplbyASET
-  - name: MK12PodIVAReplacementbyASET
-  - name: ALCOR
-  - name: ThroughTheEyesOfaKerbal
-  - name: QuickIVA
-  - name: KSAIVAUpgrade
-install:
-  - find: JSI/RasterPropMonitor
-    install_to: GameData/JSI
-  - find: JSI/RPMPodPatches/BasicMFD
-    install_to: GameData/JSI/RPMPodPatches
-    comment: >-
-      This folder contains the multi-function display props, which belong in
-      this module along with the other props
```

Comment:
Noting here in case the commit comment is missed:
Please rename to .frozen instead of deleting, the infrastructure tooling looks for this.